src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -844,4 +844,22 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_PackageDetailsUrlLink" xml:space="preserve">
     <value>Package details link</value>
   </data>
+  <data name="Label_Deprecated" xml:space="preserve">
+    <value>Deprecated</value>
+  </data>
+  <data name="Label_OrAbove" xml:space="preserve">
+    <value> (or above)</value>
+  </data>
+  <data name="Label_DeprecationReasons_CriticalBugs" xml:space="preserve">
+    <value>This package has been deprecated because it has critical bugs.</value>
+  </data>
+  <data name="Label_DeprecationReasons_Legacy" xml:space="preserve">
+    <value>This package has been deprecated because it's legacy and no longer maintained.</value>
```

Comment:
What are the recommendations around contractions (`it's` vs `it is`)? Personally, I avoid them in formal language. What do you think of this?

```suggestion
    <value>This package has been deprecated as it is legacy and no longer maintained.</value>
```

Alternatively:

```suggestion
    <value>This package has been deprecated and is no longer maintained.</value>
```

Reply:
That would also involve changing it on the gallery side, as that's where these texts are coming from.
https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/Views/Packages/_DisplayPackageDeprecation.cshtml

Reply:
@anangaur ?

Reply:
I am fine with the second suggestion "This package has been deprecated as it is legacy and no longer maintained."

Reply:
Ok. Suggestion applied. Now, what about gallery? https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/Views/Packages/_DisplayPackageDeprecation.cshtml